src/App.js

```diff
@@ -1,5 +1,123 @@
+import * as MissionUtils from "@woowacourse/mission-utils";
+
+// 3자리의 랜덤 숫자를 생성하는 함수
+const create_random_number = () => {
+  let number_list = [];
+  while (number_list.length < 3) {
+    const number = MissionUtils.Random.pickNumberInRange(1, 9).toString();
+
+    if (!number_list.includes(number)) number_list.push(number);
+  }
+
+  return number_list;
+};
+
+// 스트라이크 카운트를 계산하는 함수
+const get_strike_count = (random_number, user_number) => {
+  let count = 0;
+
+  random_number.forEach((number, index) => {
+    if (number === user_number[index]) count += 1;
+  });
+
+  return count;
+};
+
+// 볼 카운트를 계산하는 함수
+const get_ball_count = (random_number, user_number) => {
+  let count = 0;
+
+  random_number.forEach((number, index) => {
+    if (user_number.includes(number) && number !== user_number[index]) count += 1;
+  });
+
+  return count;
+};
+
+// 사용자 예측 값 입력받기
+const get_user_guess = async () => {
+  const user_input = await MissionUtils.Console.readLineAsync("숫자를 입력해주세요 : ");
+
+  // 예외 처리 ( 입력한 자리 수가 3자리 수 보다 짧은 경우, 숫자 외에 다른 문자가 입력된 경우 )
+  if (user_input == undefined || user_input.length < 3 || Number.isNaN(user_input)) {
+    throw new Error("[ERROR] 잘못된 입력입니다.");
+  }
+
+  // 예외 처리 ( 입력 값에 0이 포함된 경우, 중복된 값이 입력된 경우 )
+  const user_input_split = user_input.split("");
+  const is_includes_zero = user_input.includes("0");
+  const is_all_different_number = user_input_split.every(
+    (number) => user_input_split.indexOf(number) === user_input_split.lastIndexOf(number),
+  );
+
+  if (!is_includes_zero && is_all_different_number) return user_input;
+
+  if (is_includes_zero) {
+    throw new Error("[ERROR] 입력 값에 0이 포함되어 있습니다.");
+  } else if (!is_all_different_number) {
+    throw new Error("[ERROR] 입력 값에 중복된 값이 있습니다.");
+  } else {
+    throw new Error("[ERROR] 올바른 형식이 아닙니다.");
+  }
```

Comment:
```suggestion
  if (is_includes_zero) {
    throw new Error("[ERROR] 입력 값에 0이 포함되어 있습니다.");
  } 
  if (!is_all_different_number) {
    throw new Error("[ERROR] 입력 값에 중복된 값이 있습니다.");
  }
  throw new Error("[ERROR] 올바른 형식이 아닙니다.");
 
```

해당 부분의 가독성이 조금 떨어진다고 느꼈습니다. 문제가 없다면 해당 방식으로 고쳐볼 수 있지 않을까 생각했습니다.